.gitignore

```diff
@@ -13,4 +13,5 @@ test_code/python_scripts/testenv
 log_outputs
 /test_code/python_scripts/context
 /test_code/python_scripts/.idea
-/test_code/outputs.tf
\ No newline at end of file
+/test_code/outputs.tf
```

Comment:
Just an FYI - if these are specific to your setup, you don't have to commit these to the repo. There's a global gitignore you can control at the workstation level that usually resides in the home directory.